src/cli/index.js

```diff
@@ -171,7 +171,7 @@ if (commander.json) {
   outputWrapper = false;
 }
 if (outputWrapper) {
-  reporter.header(commandName, pkg);
+  reporter.header(commandName, {name: 'Yarn', version});
```

Comment:
Oops - I thought I ran the tests locally, but I might have only run the linter :/ I'll fix it.